src/page-objects/administration/Categories.ts

```diff
@@ -14,8 +14,6 @@ export class Categories implements PageObject {
    * Category tree
    */
   public readonly categoryTree: Locator;
-  public readonly homeCategory: Locator;
```

Comment:
saas and potentially many other systems do not have the home catgegory. We should always create all data that we expect to be in the system